stories/topics/tech-note-aws-add-remove-tags.adoc

```diff
@@ -0,0 +1,7 @@
+[id="tech-note-aws-add-remove-tags"]
+
+= Add and Remove Tags
+
+{AAPonAWS} supports stack-level tagging during stack deployment time. A stack-level tag is applied to the stack itself and all tag supported components that make up the stack.
+
+Using the add or remove tags playbooks will only update the tags for the individual components of stack and not the tag applied to the stack itself. The stack-level tags will never be modified after deploying {AAPonAWS}.
```

Comment:
playbooks only update the tags for...
Stack-level tags are never modified after deployment.